stripe_test.go

```diff
@@ -118,6 +119,71 @@ func TestDo_Retry(t *testing.T) {
 	assert.Equal(t, 2, requestNum)
 }
 
+// Tests client retries when HTTP returns timeout.
+//
+// To run only this test:
+//
+//     go test . -run TestDo_RetryOnTimeout -test.v
+//
```

Comment:
And I think this "run only this test" comment was copied from `TestDo_Retry` above. Let's just remove this part. I included it in `TestDo_Retry` because it gives you a hint at how to better understand what the test is doing, but in general there's sufficient instruction for this type of thing [in the README](https://github.com/stripe/stripe-go/issues/713).